source/_integrations/zerproc.markdown

```diff
@@ -0,0 +1,46 @@
+---
+title: Zerproc Bluetooth Lights
+description: Instructions for integrating Zerproc bluetooth lights within Home Assistant.
+ha_category:
+  - Light
+ha_iot_class: Local Polling
+ha_release: "0.111"
+ha_domain: zerproc
+---
+
+This integration discovers nearby Zerproc lights and adds them to Home Assistant.
+
+## Setup
+
+This integration requires `pybluez` to be installed. On Debian based installs, run
+
+```bash
+sudo apt install bluetooth
+```
+
+Before you get started with this integration, please note that:
```

Comment:
This is not needed for people running Home Assistant or Home Assistant Core on Docker, which is the majority of our community.

This might be confusing for non-technical users as well.